packages/core/src/utils/prepareEvent.ts

```diff
@@ -73,28 +75,27 @@ export function prepareEvent(
     }
 
     // In case we have a hub we reassign it.
-    result = finalScope.applyToEvent(prepared, hint);
+    result = finalScope.applyToEvent(prepared, hint, clientEventProcessors);
+  } else {
+    // Apply client & global event processors even if there is no scope
```

Comment:
I noticed that in some tests we actually ran into this branch, so IMHO it makes sense to ensure also global processors run if no scope is supplied (which is also kind of weird right now). This should only really be a problem when not going through `getCurrentHub().xxx`, as there we always put the scope, but e.g. if you call `captureException()` on the client directly or similar, this is not guaranteed to be there.